content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+## Objective
+
+As with the rest of the guides, this is meant as a 'hello world' example to help get you familiarized with the process of standing up a highly available Kubernetes cluster using the Replicated toolset. The architecture we will follow is presented below:
+
+![Architecture](/images/guides/kots/ha-cluster-architecture.png)
+
+The diagram shows 4 Virtual Machines (VMs) running on Google Cloud Platform (GCP), with one of them used solely as a load balancer. Note that there aren't any requirements to use GCP. You can follow along with VMs provisioned on a different cloud provider or on premise.
+
+## Overview
+
+The guide is broken into two parts. The [first part](#part-i---setting-up-the-cluster) will walk you through how to stand up the cluster, while the [second part](#part-ii-testing-scenarions--troubleshooting) will go through a couple of testing scenarios to validate the cluster's resiliency and some troubleshooting tips.
+
+There are a few things to keep in mind about this guide:
+
+- All VMs are going to be created in Google Cloud Platform (GCP) but nothing in this guide is dependent on GCP specific services. 
+- The examples in this guide used ephemeral resources.
+- The Load Balancer we'll use for this exercise is HA Proxy. 
+- The sample application has a database component as described [here](#sample-application), which will help us validate data retention in of our testing scenarios.
+
+## Prerequisites
+
+- This exercise will need 4 VMs that will need to communicate with each other. Please refer to the [Procuring the (virtual) hardware](#provisioning-the-virtual-hardware) section for guidance on system requirements for these. 
+
+- In the example commands and screenshots, we are using a sample application called 'AppDirect'. Please refer to the [Sample Application](#sample-application) section for more details if you wish to follow along and use it.
+
+- This is an advanced topic, so this guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started), and have already a level of familiarity with the Replicated tool set.
+
+
+### Sample Application
+
+The KOTS application we'll use in this guide is available in the [kotsapps repository](https://github.com/replicatedhq/kotsapps/tree/master/app-direct). 
+
+The application consists of two components:
+
+- Python Flask App Deployment - This flask application contains 'routes' to help test connecting, reading and writing to a database. 
+- Postgres StatefulSet - This is the database the for the flask application.
+
+A full description of the application is available in the repository.
+
+### Configuring Ekco
+
+If you want to follow this example, but with your own application, you may need to modify the Kubernetes Installer to use [ekco](https://kurl.sh/docs/add-ons/ekco). Ekco is basically a helper service that helps 'move' scheduled pods from a node that is down to ones that are up.
```

Comment:
I think there may be a more technically succinct way to explain what ekco does, but maybe directing folks to the ekco docs is enough. What do you think of this rephrase?

 
```suggestion
If you want to follow this example, but with your own application, you may need to modify the Kubernetes Installer to use [ekco](https://kurl.sh/docs/add-ons/ekco). For the purposes of this guide, EKCO can be thought of as a helper service that helps 'move' scheduled pods from a node that is down to ones that are up.
```

Reply:
I like that